core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java

```diff
@@ -126,6 +137,15 @@ public StackV4Response get(String crn) {
         return stackCommonService.getByCrn(crn, Collections.emptySet());
     }
 
+    @Override
+    @DisableCheckPermissions
+    @PreAuthorize("hasRole('AUTOSCALE')")
+    public WorkspaceV4Response getDefaultWorkspaceForUser(String userId, String tenant) {
```

Comment:
There should be no such endpoint, this controller will be soon removed.

Reply:
To be more precise, Workspace related API will be removed soon, so WorkspaceResponse should not be used

Reply:
ack will remove this change.